packages/shared/components/popups/VoteForProposalPopup.svelte

```diff
@@ -17,19 +17,24 @@
     )
     $: hasVotingPower = Number($selectedAccount?.votingPower) > 0
 
+    $: isTransferring = $selectedAccount?.isTransferring
+
     async function handleVoteClick(): Promise<void> {
         try {
             await checkActiveProfileAuth(async () => {
+                updateSelectedAccount({ isTransferring: true })
                 await vote($selectedAccount.index, $selectedProposal?.id, selectedAnswers)
                 showAppNotification({
                     type: 'success',
                     message: localize('notifications.vote.success'),
                     alert: true,
                 })
                 closePopup()
+                updateSelectedAccount({ isTransferring: false })
```

Comment:
This can move outside the try/catch statement to remove code duplication

Reply:
Moving outside the callback function means that we will lose the isTransferring behavior after the stronghold unlocking flow